src/common/keyboard-shortcuts.ts

```diff
@@ -0,0 +1,14 @@
+// Shortcuts are global unless noted otherwise.
+export const keyboardShortcuts = {
+  // This is scoped by keyboard focus.
+  copyCode: ["ctrl+c", "meta+c", "enter"],
+  search: ["ctrl+shift+f", "meta+shift+f"],
+  sendToMicrobit: ["ctrl+shift+e", "meta+shift+e"],
+  saveProject: ["ctrl+shift+s", "meta+shift+s"],
```

Comment:
Keybindings to discuss. I assumed we wanted a shortcut for save project rather than save file, but I suppose we could have both if we wanted.

Reply:
Yep we really want to encourage the hex to be what gets saved so that makes sense to me.